browser/react/components/Yoonah.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { joinChatRoom, leaveChatRoom } from '../../webRTC/client.js';
+import '../../aframeComponents/scene-load';
+import '../../aframeComponents/aframe-hyperlink';
+
+export default class Yoonah extends React.Component {
+
+  componentDidMount () {
+    console.log('joining the beth chat room');
+    joinChatRoom('yoonah');
+  }
+
+  componentWillUnmount () {
+    console.log('leaving the yoonah chat room');
+    leaveChatRoom('yoonah');
+  }
+
+  render () {
+    return (
+      <a-entity id="room" class="yoonah" position="0 0 0">
+        <a-entity position="-10 1 -1">
+          <a-entity class='clickable' geometry="primitive: sphere; radius:0.3;" rotation="0 200 0" material="color: green; opacity: 1; roughness: 0.8" href="/vr"></a-entity>
```

Comment:
Sure, that sounds like a good idea. Looking at it, I'm thinking <a-teleporter> which takes `color`, `href`, and `position`.